python_modules/dagster/dagster/_core/execution/build_resources.py

```diff
@@ -43,13 +43,13 @@ def build_resources(
     resources: Mapping[str, Any],
     instance: Optional[DagsterInstance] = None,
     resource_config: Optional[Mapping[str, Any]] = None,
-    pipeline_run: Optional[DagsterRun] = None,
+    dagster_run: Optional[DagsterRun] = None,
```

Comment:
[1] public API so can't delete arg directly here - let's keep both around until 1.2 and then delete

Reply:
👍